src/AuditReader.php

```diff
@@ -774,8 +775,12 @@ protected function getEntityPersister($className)
     /**
      * Simplified and stolen code from UnitOfWork::createEntity.
      *
-     * @param string     $className
-     * @param int|string $revision
+     * @template T of object
+     *
+     * @param string                $className
```

Comment:
isnt this a class-string?

Reply:
yep, all these parameters are defined below as `@phpstan-param`